src/main.rs

```diff
@@ -34,12 +35,8 @@ const PREFS_KEY: &str = "config/gui";
 const PREFS_KEY_SERIAL: &str = "config/serial_devices";
 
 fn split(payload: &str) -> Vec<f32> {
-    let mut split_data: Vec<&str> = vec![];
-    for s in payload.split(':') {
-        split_data.extend(s.split(','));
-    }
-    split_data
-        .iter()
+    payload
+        .split(&[':', ',', '=', ' ', '\t'])
```

Comment:
This was just to get the output of ping to look nice in the screenshot. I am happy to revert this if you want